src/main/java/pl/edu/mimuw/queue/AbstractIntQueue.java

```diff
@@ -2,38 +2,71 @@
 
 public abstract class AbstractIntQueue {
 
-  // TODO: you can make changes with this class fields, constructors
-  //  but also add some methods but the specified methods cannot be changed (you
-  //  can change them not to be abstract and provide some implementation for them,
-  //  but they have to have the same names, arguments and returned values)
+  /**
+   * front to zawsze reprezentacja elementu, ktory zostanie pierwszy zdjety z kolejki lub null gdy jest pusta
```

Comment:
ja zacząłem po angielsku, to pisz proszę po angielsku albo wszystko na polski przepisz - dbamy o spójność tego co piszemy 😉